src/test/java/core/web/DeleteDashboardTest.java

```diff
@@ -0,0 +1,48 @@
+package core.web;
+
+import core.driver.RunType;
+import core.driver.WebDriverHolder;
+import core.model.User;
+import core.web.pageObjects.DashboardPage;
+import core.web.pageObjects.LoginPage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class DeleteDashboardTest extends BaseTest {
+
+    @Test
+    @org.testng.annotations.Test
+    public void deleteFirstDashboardTest() {
+        final String homePageURL = webConfiguration.getRunType() == RunType.LOCAL ?
+                webConfiguration.getLocalUrl() :
+                webConfiguration.getRemoteUrl();
+        User user = User.createUser();
+        LoginPage loginPage = new LoginPage(WebDriverHolder.getInstance().getWebDriver());
+        DashboardPage dashboardPage = new DashboardPage(WebDriverHolder.getInstance().getWebDriver());
+
+        loginPage
+                .openPage(homePageURL)
+                .typeLogin(user.getLogin())
+                .typePassword(user.getPassword())
+                .clickLoginButton();
+
+        List<WebElement> dashboardsBeforeDeleting = dashboardPage.getDasboardsList();
+
+        List<WebElement> dashboardsAfterDeleting = dashboardPage
+                .clickDeleteFirstDashboardButton()
```

Comment:
This test depends on the test data. Let's create an API pre-condition that creates a dashboard with a randomly generated name. In the test, delete the dashboard by name. Add teardown, which guarantees that the dashboard is deleted (try to delete it using API)